data/json/recipes/recipe_food.json

```diff
@@ -6345,7 +6385,33 @@
     "autolearn": true,
     "batch_time_factors": [ 83, 3 ],
     "tools": [ [ [ "food_processor", 20 ] ] ],
-    "components": [ [ [ "corn", 1 ], [ "irradiated_corn", 1 ], [ "kernels", 1 ] ] ]
+    "components": [ [ [ "dry_corn", 1 ] ] ]
+  },
+  {
+    "result": "corn_kernels",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "byproducts": [ [ "empty_corn_cob", 1 ] ],
+    "skill_used": "cooking",
+    "time": "20 s",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "corn", 1 ] ] ]
+  },
+  {
+    "result": "irradiated_corn_kernels",
+    "type": "recipe",
```

Comment:
Once one starts cutting into an irradiated (and then sealed!) vegetable, one has introduced potential spoilage bacteria. I do not think this would give you anything other than regular corn kernels and an empty cob.

Otherwise, looking good!

Reply:
Good call, I reverted those changes.